pkg/ibmcsidriver/controller_helper.go

```diff
@@ -374,6 +415,47 @@ func overrideParams(logger *zap.Logger, req *csi.CreateVolumeRequest, config *co
 					}
 				}
 			}
+		case SecurityGroupIDs:
+			if len(value) != 0 {
+				securityGroupstr := strings.TrimSpace(value)
```

Comment:
I see this code redundant, can we have a common function, which takes the value string and returns securityGroupList.

Reply:
Ok I had not added this code in secrets earlier so had kept this pending until we understand the functionality will update this soon